lib/pbench/test/functional/server/test_audit.py

```diff
@@ -0,0 +1,44 @@
+from pbench.client import API, PbenchServerClient
+
+
+class TestAudit:
+    def test_get_all(self, server_client: PbenchServerClient, login_admin):
```

Comment:
Would it work to add a `@pytest.mark.dependency(depends=["upload"])` here, to ensure that there is something to audit?  (I don't know what the scope of the dependency list is.)

Reply:
Sadly, no; that only works within a file. When I added dependencies, I was *hoping* that would allow me to break up the increasingly massive `test_put.py` into `upload` and `list` and `publish` and so forth, using cross-file dependencies: but the package I found doesn't support that and I wasn't able to find another that did.

At the same time, I really don't want to add this to the `test_datasets.py` file.

I think that, in normal sequential execution, `pytest` alphabetizes the files and we might be able to "fake it" with `test_001_connect.py`, `test_002_datasets.py`, and so forth; however I don't think that default rule holds if we want to try running parallel test cases, which will probably be valuable as our functional test suite expands.